go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/nanmu42/etherscan-api
 
-go 1.13
+go 1.19
```

Comment:
Since we are not using latest go features, keeping the go version old would be better for our existing users.